eox_core/api/v1/views.py

```diff
@@ -54,6 +54,15 @@ def __init__(self, *args, **kwargs):
         self.query_params = None
         self.site = None
 
+    def initial(self, request, *args, **kwargs):
```

Comment:
who calls this method?

Reply:
the dispatch method in DRF

Reply:
Just a minor comment:

I did not understand how `super()` was working in this context. Here two references to check:
https://rhettinger.wordpress.com/2011/05/26/super-considered-super/
https://stackoverflow.com/questions/27954695/what-is-a-sibling-class-in-python

`super(UserQueryMixin, self).initial(request, *args, **kwargs)` is a call for [DRF initial method defined in views ](https://github.com/encode/django-rest-framework/blob/afb678433b7dc068213e6e8a359ad1f6fff05b0d/rest_framework/views.py#L387) since it's the sibling class.

cc: @felipemontoya @jfavellar90 @Squirrel18

Reply:
@diegomillan just checked using ipdb and the DRF method is getting called. So it is working as it should. I'm closing this then

Reply:
Yes, it is working @felipemontoya. I just wanted to explain the reason why,  if anyone wants to check. 
LGTM this is approved.